quartz/components/scripts/callout.inline.ts

```diff
@@ -2,23 +2,17 @@ function toggleCallout(this: HTMLElement) {
   const outerBlock = this.parentElement!
   outerBlock.classList.toggle("is-collapsed")
   const collapsed = outerBlock.classList.contains("is-collapsed")
-  const height = collapsed ? this.scrollHeight : outerBlock.scrollHeight
-  outerBlock.style.maxHeight = height + "px"
 
-  // walk and adjust height of all parents
-  let current = outerBlock
-  let parent = outerBlock.parentElement
-  while (parent) {
-    if (!parent.classList.contains("callout")) {
-      return
-    }
-
-    const collapsed = parent.classList.contains("is-collapsed")
-    const height = collapsed ? parent.scrollHeight : parent.scrollHeight + current.scrollHeight
-    parent.style.maxHeight = height + "px"
-
-    current = parent
-    parent = parent.parentElement
+  // Check if the block is not collapsed
+  if (!collapsed) {
```

Comment:
I pulled this in and it works for me
![Screenshot 2024-11-10 at 8 42 03 PM](https://github.com/user-attachments/assets/00301a4a-c96d-4463-84db-ba689013c4a5)